src/utils/canvas-shim.js

```diff
@@ -0,0 +1,28 @@
+import CanvasRenderer from '../renderers/canvas';
```

Comment:
This style of imports (without the `.js` suffix) fails in at least Chrome. Is there a reason to not have the full filename?

See this screenshot of the console:

![Capture](https://user-images.githubusercontent.com/220033/73402359-6b1d8300-42ed-11ea-8ad6-e146bc005bd2.PNG)

Reply:
You can test this in the browser by adding `type="module"` to your script tag like so:

````javascript
<script type="module">
  import Two from '../src/two.js';
</script>
````

Reply:
I was originally only targeting Rollup and didn't think to test this. I'll add `.js` to the imports to they work as-is in the browser as well.